packages/framework-core/src/entity-destroyer.ts

```diff
@@ -0,0 +1,12 @@
+import { UUID, BoosterConfig, EntityInterface, Logger, Class } from '@boostercloud/framework-types'
+import { EventStore } from './services/event-store'
+
+export async function destroyEntity<TEntity extends EntityInterface>(
+  config: BoosterConfig,
+  logger: Logger,
+  entityClass: Class<TEntity>,
+  entityID: UUID
+): Promise<void> {
+  const eventStore = new EventStore(config, logger)
```

Comment:
It's weird that you have to create an event store instance when there's a single event store in the whole system. We should eventually review this implementation detail...

Reply:
@javiertoledo Agreed, I think we should create the event store at a Booster main class level and call it directly. In that way we could remove the `entity-detroyer.ts` and also de `entity-snapshot-fetcher.ts`. Makes sense?